src/modules/redesign/i18n/fr_FR/alliance_avatar.json

```diff
@@ -0,0 +1,7 @@
+{
+	"current": "Current profile picture of the alliance",
+	"preview": "Preview",
+	"save": "Save",
+	"select": "Select image from computer",
+	"title": "Edit Alliance Logo"
+}
```

Comment:
translation?

Reply:
I have created all the files.
I translate them, sorry
Translation ok